src/Kdyby/Doctrine/DI/OrmExtension.php

```diff
@@ -288,7 +288,7 @@ protected function processEntityManager($name, array $defaults)
 			$metadataDriver->addSetup('setDefaultDriver', [
 				new Statement($this->metadataDriverClasses[self::ANNOTATION_DRIVER], [
 					'@' . Doctrine\Common\Annotations\Reader::class,
-					[$builder->expand('%appDir%')]
+					[Nette\DI\Helpers::expand('%appDir%', $this->compiler->getContainerBuilder()->parameters)]
```

Comment:
Is `$this->compiler->getContainerBuilder()` different from `$builder`?

Reply:
no, it's same... fixed.